docs/vm-installation.md

```diff
@@ -0,0 +1,78 @@
+# Deploying antrea-agent on VM
+
+Antrea-agent can be run on Linux/Windows VMs. Below are the steps needed to configure and run antrea-agent on VMs
+
+## Prerequisites on cluster
+1. Create a name-space for the antrea agent
```

Comment:
Namespace

the antrea agent -> `antrea-agent`

Reply:
Done